test_src/sljitTest.c

```diff
@@ -11913,7 +11913,7 @@ static void test94(void)
 	struct sljit_compiler* compiler;
 	sljit_f64 dbuf[9];
 	sljit_f32 sbuf[9];
-	sljit_s32 i;
+	int i;
```

Comment:
Why do we need this change?

Reply:
It is easier to read and more common, but you are correct it isn't needed